charts/hawkbit-update-server/values.yaml

```diff
@@ -108,3 +86,26 @@ rabbitmq:
     password: hawkbit
     metrics:
       enabled: true
+
+extraVolumes: []
+extraVolumeMounts: []
+
+configMap:
+  mountPath: "/opt/hawkbit/config"
+
+spring:
+  profiles: "mysql"
+config:
+  secrets:
+    spring:
+      security:
+        user:
+          password: "{noop}admin"
```

Comment:
still don't like the encoder in the password string.
should be separate variable.

Reply:
I think this is a "feature" of spring and its ok to use it as it is ... changing this will make the configuration more complex

Reply:
i disagree as most people would likely never touch the additional variable but are less confused by the "{noop}" prefix.